CHANGELOG.rst

```diff
@@ -1,5 +1,51 @@
-Will be updated by antsibull-changelog. Do not edit this manually!
+===============================
+Cloud.VMware\_Ops Release Notes
+===============================
 
-See https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst for information on how to use antsibull-changelog.
+.. contents:: Topics
 
-Check out ``changelogs/config.yaml`` for its configuration. You need to change at least the ``title`` field in there.
+v1.2.0
+======
+
+Major Changes
```

Comment:
This is not a major change... I will open a dedicated PR to fix it... and re-run the changelog command

Reply:
merged that pr and re-generated the changelog